wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -454,6 +450,14 @@ public extension UserClient {
             selfUser.needsToAcknowledgeLegalHoldStatus = true
         }
 
+        if !client.isSelfClient() {
```

Comment:
why don't we pass the keys on selfClient?

Reply:
Because we send them to the backend, it never makes sense to retrieve then from the backend that only opens up for things going out of sync.

Reply:
makes sense